### ISSUE TITLE
Some examples go into reboot (e.g. example 05-WorkingLED) 🥹

### ISSUE THREAD

**Issue by exrector** [HomeSpan/HomeSpan#876]
Some examples (01,02,16) work fine and you can press W to select the network, but some examples (05, 14) go into reboot after pressing W
`*** WiFi Setup - Scanning for Networks...

Guru Meditation Error: Core  1 panic'ed (StoreProhibited). Exception was unhandled.

Core  1 register dump:
PC      : 0x400dcdb6  PS      : 0x00060730  A0      : 0x800dce6c  A1      : 0x3ffb1f70  
A2      : 0x3ffc4aec  A3      : 0x00000000  A4      : 0x00000000  A5      : 0x3ffc544c  
A6      : 0x00000000  A7      : 0x0000000b  A8      : 0x800dcdb6  A9      : 0x3ffb1f50  
A10     : 0x00000000  A11     : 0x00000001  A12     : 0x00000000  A13     : 0x00000000  
A14     : 0x0000012c  A15     : 0x00000000  SAR     : 0x0000000a  EXCCAUSE: 0x0000001d  
EXCVADDR: 0x00000000  LBEG    : 0x4008c941  LEND    : 0x4008c951  LCOUNT  : 0xfffffffd  

Backtrace: 0x400dcdb3:0x3ffb1f70 0x400dce69:0x3ffb1fd0 0x400dadce:0x3ffb2010 0x400dc5e9:0x3ffb2170 0x400dcc13:0x3ffb2250 0x400d32da:0x3ffb2270 0x400e56c1:0x3ffb2290

ELF file SHA256: 38733f29f755ee41

Rebooting...
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x33 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0030,len:1344
load:0x40078000,len:13964
load:0x40080400,len:3600
entry 0x400805f0

************************************************************
Welcome to HomeSpan!
Apple HomeKit for the Espressif ESP-32 WROOM and Arduino IDE
************************************************************
** Please ensure serial monitor is set to transmit <newlines>

Message Logs:     Level 0
Status LED:       Pin -  *** WARNING: Status LED Pin is UNDEFINED
Device Control:   Pin -  *** WARNING: Device Control Pin is UNDEFINED
Sketch Version:   n/a
HomeSpan Version: 1.9.0
Arduino-ESP Ver.: 2.0.17
ESP-IDF Version:  4.4.7
ESP32 Chip:       ESP32-D0WDQ6 Rev 1 dual-core 4MB Flash
ESP32 Board:      esp32
PWM Resources:    16 channels, 8 timers, max 20-bit duty resolution
Sodium Version:   1.0.12-idf  Lib 9.4
MbedTLS Version:  Mbed TLS 2.28.7
Sketch Compiled:  Jul  2 2024 14:48:58
Partition:        app0
MAC Address:      3C:71:BF:5E:EA:1C

Device Name:      HomeSpan LED

*** HomeSpan Info ***
➤ Accessory:  AID=1
   ➟ Service AccessoryInformation:  IID=1, UUID="3E"
      ⇨ Characteristic Identify(1):  IID=2, UUID="14", Perms=PW
   ➟ Service LightBulb:  IID=3, UUID="43"
      ⇨ Characteristic On(0):  IID=4, UUID="25", Perms=PR+PW+EV
➤ Accessory:  AID=2
   ➟ Service AccessoryInformation:  IID=1, UUID="3E"
      ⇨ Characteristic Identify(1):  IID=2, UUID="14", Perms=PW
   ➟ Service LightBulb:  IID=3, UUID="43"
      ⇨ Characteristic On(0):  IID=4, UUID="25", Perms=PR+PW+EV

Service                             UUID         AID  IID  Update  Loop  Button  Linked Services
------------------------------  --------  ----------  ---  ------  ----  ------  ---------------
AccessoryInformation                  3E           1    1      NO    NO      NO  -
LightBulb                             43           1    3     YES    NO      NO  -
AccessoryInformation                  3E           2    1      NO    NO      NO  -
LightBulb                             43           2    3     YES    NO      NO  -

Configured as Bridge: NO

Database Validation:  Warnings=0, Errors=0

*** End Info ***

Accessory ID:      26:77:EA:C7:4A:1F                               LTPK: D60AEBBCEF3E998263AE4B4DB2931F29F5F794ECD0BE4CFB6C31E7BB72AD2927
No Paired Controllers

Accessory configuration number: 3

*** WIFI CREDENTIALS DATA NOT FOUND.  YOU MAY CONFIGURE BY TYPING 'W <RETURN>'.

HomeSpan LED is READY!
`

**Comment by HomeSpan** [comment]
Did you alter the pin numbers in example 5, or otherwise make any changes at all to the sketch? Also, are you using the original ESp32 or a different chip?

**Comment by exrector** [comment]
> Вы меняли номера выводов в примере 5 или вносили какие-либо изменения в скетч? ​​Кроме того, вы используете оригинальный ESp32 или другой чип?

I did nothing. I  just opened the sketch, compiled and flashed the board

**Comment by exrector** [comment]
I found the problem. Thanks for the tip. I used the "wrower" board and selected "wrower" in arduino boards manager.
And it doesn't work.
Then I took the "wroom" board and set "wroom" in arduino boards manager.
It works great
Then I left the "wroom" in arduino boards manager and connected the "wrower" board. Flashed the device.
And it WORKS.
(apparently the problem lies in the library of supported boards).
Namaste. Thank you. And good luck!
